dotnet/samples/Demos/CodeInterpreterPlugin/README.md

```diff
@@ -31,3 +31,21 @@ OpenAI__ChatModelId
 # Azure Container Apps
 AzureContainerApps__Endpoint
 ```
+
+### Usage Example
+
+User: Upload the file c:\temp\code-interpreter\test-file.txt
+
+Assistant: The file test-file.txt has been successfully uploaded.
+
+User: How many files I have uploaded ?
+
+Assistant: You have uploaded 1 file.
+
+User: Show me the contents of this file
```

Comment:
Do we need to use the plural here to refer to the file's content?

Reply:
Not necessarily, this was just my prompt, could be in the singular or even with typo, the AI would catch it